src/components/Modals/Nfts/NftModal.tsx

```diff
@@ -283,18 +284,14 @@ export const NftModal: React.FC<NftModalProps> = ({ nftItem }) => {
         </Box>
         <TabPanels maxHeight={{ base: 'auto', md: '500px' }} overflowY='auto' flex={1}>
           <TabPanel p={0}>
-            <NftOverview nftItem={nftItem} nftCollection={nftCollection} />
+            <NftOverview nftItem={nftItem} />
           </TabPanel>
           {hasUsefulCollectionData && (
             <TabPanel p={0}>
               <NftCollection
-                name={nftCollection?.name || nftItem.collection.name}
-                description={nftCollection?.description || nftItem.collection.description}
-                socialLinks={
-                  nftCollection?.socialLinks.length
-                    ? nftCollection?.socialLinks
-                    : nftItem.collection.socialLinks
-                }
+                name={nftCollection.name}
+                description={nftCollection?.description}
```

Comment:
one more optional chain that can be removed

Reply:
7c0b1b6df